src/CoreBundle/Resources/config/services.yml

```diff
@@ -1,4 +1,9 @@
 services:
+    MetaModels\IFactory: '@metamodels.factory'
+    MetaModels\Filter\Setting\FilterSettingFactory: '@metamodels.filter_setting_factory'
```

Comment:
Shouldn't this be:
```diff
- MetaModels\Filter\Setting\FilterSettingFactory: '@metamodels.filter_setting_factory'
+ MetaModels\Filter\Setting\IFilterSettingFactory: '@metamodels.filter_setting_factory'
- MetaModels\Render\Setting\RenderSettingFactory: '@metamodels.render_setting_factory'
+ MetaModels\Render\Setting\IRenderSettingFactory: '@metamodels.render_setting_factory'
```
as these are also interface

Reply:
What more interfaces? `IItem`and `IItems` should not be injected and `IMetaModelsServiceContainer` is deprecated.